lib/streamwrapper.php

```diff
@@ -614,7 +616,7 @@ private function getOption($name) {
 	/**
 	 * Gets the client from the stream context
 	 *
-	 * @return S3ClientInterface
+	 * @return S3Client
```

Comment:
`getClient()` does actually return a full `S3Client` which has more methods than just those in `S3ClientInterface`
Adjusting the PHP doc here stops `phpstan`  reporting:
```
php -d zend.enable_gc=0 vendor-bin/phpstan/vendor/bin/phpstan analyse --memory-limit=4G --configuration=./phpstan.neon --no-progress --level=5 appinfo lib
 ------ ----------------------------------------------------------------------- 
  Line   lib/streamwrapper.php                                                  
 ------ ----------------------------------------------------------------------- 
  175    Call to an undefined method Aws\S3\S3ClientInterface::putObject().     
  207    Call to an undefined method Aws\S3\S3ClientInterface::deleteObject().  
  266    Call to an undefined method Aws\S3\S3ClientInterface::headObject().    
  280    Call to an undefined method Aws\S3\S3ClientInterface::listObjects().   
  347    Call to an undefined method Aws\S3\S3ClientInterface::deleteBucket().  
  526    Call to an undefined method Aws\S3\S3ClientInterface::deleteObject().  
  682    Call to an undefined method Aws\S3\S3ClientInterface::getObject().     
  765    Call to an undefined method Aws\S3\S3ClientInterface::createBucket().  
  793    Call to an undefined method Aws\S3\S3ClientInterface::putObject().     
  810    Call to an undefined method Aws\S3\S3ClientInterface::listObjects().   
 ------ ----------------------------------------------------------------------- 

                                                                                                                        
 [ERROR] Found 10 errors                                                                                                
                                                                                                                    
```